pkgs/applications/editors/kakoune.cr/default.nix

```diff
@@ -25,16 +25,11 @@ crystal.buildCrystalPackage rec {
   lockFile = ./shard.lock;
 
   installPhase = ''
```

Comment:
You should copy `bin` and `share` to `$out`.

Reply:
> @loewenheim It is important to copy the whole `bin` and `share` directories to `$out`, as `kcr` will look in `share`.

Do the various scripts in `share` still need to be linked into `bin`?

Reply:
If you want `nix-shell -p kakoune.cr` and have the shipped commands in your path yes.

Otherwise, Nix users will need to enter the following commands:

``` sh
kcr install commands
kcr install desktop
```

Reply:
The idea is definitely that you don't have to run these commands after installing with `nix`.